content/02.introduction.md

```diff
@@ -10,66 +10,62 @@ One week later, 4593 confirmed cases had been identified, spanning not only Asia
 On March 11, 2020, WHO formally classified the situation as a pandemic [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf].
 On April 4, 2020, the WHO reported that the global number of confirmed cases had surpassed one million [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200404-sitrep-75-covid-19.pdf?sfvrsn=99251b2b_4].
 At this time, over 92,000 deaths had been reported due to COVID-19 worldwide.
+New information is emerging daily about the impact of the disease around the world.
 
-[Note: Maybe add a graph here, update as new reports come out.]
-
-COVID-19 is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).
-SARS-CoV-2 is a coronavirus, a family of RNA viruses known to cause respiratory and intestinal infections in humans and other species.
+COVID-19 is caused by the newly identified severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which was initially called 2019 novel Coronavirus (19-nCov).
```

Comment:
At the end will we stick to only one abbreviation of the current virus? I remember using COVID-19, others using SARS-CoV-2 etc.  Although I guess COVID-19 refers to the infection itself rather than the virus for which SAR-CoV-2 denotes, so my point is probably invalid.

Reply:
I think you're right on the SARS-CoV-2 being the virus and COVID being the disease. Your proposals sound good :)